nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/FileSystemRepository.java

```diff
@@ -509,52 +508,52 @@ public ContentClaim create(final boolean lossTolerant) throws IOException {
         // action to ensure that the comparison of those two conditions is atomic also. As a result,
         // we will synchronize on the queue while performing those actions.
```

Comment:
Since synchronization is now done in StandardResourceClaimManager, I think this long comment no longer applies.  Leaving it in might confuse our future selves.